### ISSUE TITLE
Transformation Techniques on SVM

### ISSUE THREAD

**Issue by avagreyyy4** [mikeizbicki/cmc-csci145-math166#207]
Hi! I understand we have learned that centering or scaling the data has no effect on the VC dimension since it is not impacting the dimensionality. However, for the SVM where we now have a dvc that relies on the radius and rho, would centering the data, for example, impact the dvc since it is decreasing the radius value? Thanks

**Comment by mikeizbicki** [comment]
You're correct that centering will reduce the bound on the "vc dimension" for SVM provided by Theorem 8.5 in the textbook.  But you are also responsible for understanding why there are scare quotes around the phrase VC dimension above.
